articles/quickstart/native/android-vnext/01-user-metadata.md

```diff
@@ -114,7 +114,7 @@ profile?.getId()?.let { userId ->
           override fun onSuccess(profile: UserProfile?) {
               // The metadata was updated and we're given the updated user profile.
               // Retrieve the "country" field, if one appears in the metadata
-              val country = profile?.getUserMetadata()?.get("country") as String? ?: ""
+              val country: String? = profile?.getUserMetadata()?.get("country")
```

Comment:
this line would need the cast (after what we just resolved). Mind if this is repeated somewhere else in this file.